frontend/src/hooks/useSessionValid.js

```diff
@@ -133,18 +135,14 @@ function useSessionValid() {
       // Set the session details
       setSessionDetails(getSessionData(userAndOrgDetails));
     } catch (err) {
-      // TODO: Throw popup error message
```

Comment:
why id we remove this piece of code? @chandrasekharan-zipstack

Reply:
@johnyrahul I've added it after the condition that checks for the request status of 412. Previously we weren't raising an alert

Reply:
Thanks for pointing this out @johnyrahul 
@vishnuszipstack @tahierhussain can you validate this behaviour for subscription period expiry? I've reverted the older implementation but please confirm if its valid to check that for all 402 status codes

Reply:
@chandrasekharan-zipstack LGTM. @tahierhussain Please confirm.

Reply:
Discussed with @tahierhussain to move the navigation to trial expiry page here however it didn't work as expected. Leaving the old code as it is with a comment to review the behaviour later